### PR TITLE
Bump Symfony versions tests to 2.8

### LINKED ISSUE #51: Next major: Symfony version support
Let's talk about Symfony version support drop for next major of sonata bundles.

I think we all agree that SF 2.3 to 2.6 would be dropped.

What I'm not sure it's about 2.7. Should will still support it?

My thinking: Keeping only 2.8+ support would get us free of **LOT** of BC tricks between SF 2.7 and 2.8, especially for Form.

With that, we will have a clean code to start BC layers for next 3.x versions of Symfony.

I know, that 2.7 is a LTS version. But this one is supported by our actual bundles and SF 2.8 is also an LTS. BTW, 2.8 is here because of a too early release of 2.7 AFAIK.

Let's vote, use emoji reaction for that:

:+1: For support beginning at SF **2.8**
:-1: For support beginning at SF **2.7**

Feel free to elaborate your thinking on this issue. :+1: 

cc @sonata-project/contributors

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Closes #51.

This will update Travis config **only**. Modifications have to be made on the `composer.json` of each project too.

@sonata-project/contributors If you have any suggestion/question, feel free! :+1:

### CODE REVIEW

**Comment by core23** [comment]
Can we merge this @Soullivaneuh ?

**Comment by OskarStark** [comment]
But we currently support Symfony 2.3, or not?

**Comment by core23** [comment]
Yes, the stable still supports 2.3. This affects only the master branches.

**Comment by core23** [comment]
Ping @Soullivaneuh ?

**Comment by core23** [comment]
Can we merge this @Soullivaneuh ?

**Comment by greg0ire** [comment]
If you want @Soullivaneuh to answer on a particular post, I think you should talk to him on slack.

**Comment by greg0ire** [comment]
> This affects only the master branches.

Or does it?

File: config/projects.yml [hunk 1]
```diff
@@ -259,7 +259,7 @@ media-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0']
       target_php: 5.6
       versions:
-        symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
+        symfony: ['2.8', '3.0', '3.1']
```
Comment (greg0ire) [inline config/projects.yml]: @jordisala1991 : the mistake happened here. Can you do a PR on the admin bundle, just to see if the build would be green if we restored those?

File: config/projects.yml [hunk 2]
```diff
@@ -259,7 +259,7 @@ media-bundle:
...
         doctrine_odm: ['1']
         sonata_core: ['3']
         sonata_admin: ['3']
```
Comment (greg0ire) [inline config/projects.yml]: Also, not sure what is the point of these lines…

Comment (jordisala1991) [inline config/projects.yml]: This line add a build on travis to check if we are compatible with 3.x

Comment (greg0ire) [inline config/projects.yml]: Yeah, so… what's the point? 3.x is the default I think, and is used on almost every other build

Comment (jordisala1991) [inline config/projects.yml]: when you build media-bundle 3.x you are using a tag of sonata-admin like 3.13.0. This allows to test against sonata-admin 3.x

Comment (greg0ire) [inline config/projects.yml]: Oh right…

**Pull request merged**